pkgs/desktops/gnome/extensions/pano/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  glib,
+  libgda6,
+  gsound,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gnome-shell-extension-pano";
+  version = "23-alpha5";
+
+  src = fetchzip {
+    url = "https://github.com/oae/gnome-shell-pano/releases/download/v${finalAttrs.version}/pano@elhan.io.zip";
+    hash = "sha256-kTaJOSyFtBa/fl3Mot8Q8qyhwJwhcbBY4FvdztqUP4w=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    glib
+  ];
+
+  buildPhase = ''
+    glib-compile-schemas --strict schemas
```

Comment:
When overriding `buildPhase`, you should also provide `runHook preBuild` and `runHook postBuild`. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md> for more info.